jerry-core/debugger/debugger.c

```diff
@@ -468,6 +473,7 @@ jerry_debugger_process_message (const uint8_t *recv_buffer_p, /**< pointer to th
     {
       JERRY_DEBUGGER_CHECK_PACKET_SIZE (jerry_debugger_receive_type_t);
 
+      JERRY_INIT_CURRENT_INSTANCE ()
```

Comment:
AFAIK: the macro is already used at the start of the method. Do we still need this here?